src/enums/SpecialKeys.ts

```diff
@@ -10,4 +10,6 @@ export enum SpecialKeys {
   ARROW_DOWN = '\uE015',
   TAB = '\uE004',
   ESCAPE = '\uE00C',
+  EMOJI_HEART = '❤', //In The BMP (basic multilingual plane)
```

Comment:
please check, that ci machine(linux) can read this emojies

Reply:
fixed and changed to unicode